src/katsdpcalproc/pointing.py

```diff
@@ -345,56 +338,67 @@ def calc_pointing_offsets(session, beams, target, middle_time,
           - rough uncertainty (standard deviation) of (az, el) adjustment.
 
     """
+
+    if isinstance(existing_az_el_adjust, type(None)):
+        existing_az_el_adjust = np.zeros((len(ants), 2))
+    if middle_time < 16000000:
+        raise NotUnixTime("Middle times must be in unix time format")
+    if not isinstance(target, katpoint.Target):
```

Comment:
Same here. Let the function crash if the type is wrong.

Reply:
Removed these custom errors